### PR TITLE
Update/dependencies composables

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: components/Crs/Selector.vue [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  }
+}
+
+async function get_crs_table () {
+  let params = new FormData()
+  console.log('geode_object', geode_object)
+  params.append('geode_object', geode_object)
+  const route = `/geographic_coordinate_systems`
+  toggle_loading()
+  await api_fetch(route, { method: 'POST', body: params },
```
Comment (BotellaA) [inline components/Crs/Selector.vue:59]: POST or GET?

File: components/Left/CoordinateSystems/index.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+const id = ref(0)
+const coordinate_systems = ref([])
+
+async function get_coordinate_systems () {
+  ws_link_store.$patch({ busy: true })
+  const params = new FormData()
+  params.append('native_file_name', native_file_name)
+  params.append('geode_object', geode_object)
+  await api_fetch(`/coordinate_systems`, { body: params, method: 'POST' }, {
```
Comment (BotellaA) [inline components/Left/CoordinateSystems/index.vue:72]: GET ?

File: components/Crs/Assign/Button.vue [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<template>
+  <v-btn :loading="loading" color="primary" @click="convert_files()">
+    Assign
+    <template #loader>
+      <v-progress-circular indeterminate size="20" color="white" width="3" />
+    </template>
+  </v-btn>
+  <v-btn variant="text" @click="current_step = 3">
```
[the remark this answers is not in the document]
Comment (JulienChampagnol) [reply components/Crs/Assign/Button.vue:8]: Go back to previous step, it should be 'stepper_tree.current_step_index = 3' here

File: components/Left/CoordinateSystems/index.vue (hunk 2 above)
Comment (JulienChampagnol) [reply components/Left/CoordinateSystems/index.vue:72]: Same

**Review by BotellaA — APPROVED** [review]

**Pull request merged**